app/src/main/java/com/codesoom/assignment/errors/UserNotFoundException.java

```diff
@@ -1,5 +1,8 @@
 package com.codesoom.assignment.errors;
 
+/**
+ * 사용자를 찾지 못할 경우 발생되는 예외 클래스
```

Comment:
"발생되는"은 적절한 표현이 아닙니다. 마치 어디서 우리가 모르는 사람이 예외를 던지는 것 같잖아요. 

```suggestion
 * 사용자를 찾지 못할 경우 던집니다.
```